web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -680,12 +715,16 @@
         contentType: contentType
       });
     },
-    ajaxGet: function(url, data, onSuccess, onFailure) {
+    ajaxGet: function(url, data, onSuccess, onFailure, dataType) {
+      if (!dataType) {
+        dataType = "script";
+      }
+
```

Comment:
@Shadowfiend This is the change I was talking about in this ML thread https://groups.google.com/forum/#!topic/liftweb/KdH4kgZ-5ug

Reply:
Because Javascript, this change is backward compatible. I don't think we have to worry too much about this change so long as the `dataType` check works as we expect.